kalkulator.py

```diff
@@ -0,0 +1,31 @@
+OPERATIONS = ["+", "-", "*", "/", "%"]
+
+def check_operation(math_operation: str) -> bool:
+    if math_operation not in OPERATIONS:
+        return True
+    else:
+        return False
```

Comment:
No need to have else. :) 
```suggestion
def check_operation(math_operation: str) -> bool:
    if math_operation not in OPERATIONS:
        return True
    return False
```